clients/restclient/rest_client.go

```diff
@@ -43,3 +50,23 @@ func (c RestClient) PurgeConfiguration(org, space string) error {
 	}
 	return nil
 }
+
+func (c RestClient) getCsrfToken() error {
+	_, err := executeRestOperation(c.TokenFactory, func(token runtime.ClientAuthInfoWriter) (interface{}, error) {
+		return c.Client.Operations.GetCsrfToken(nil, token)
+	})
+	return err
+}
+
+func (c RestClient) GetSession() error {
+	c.getCsrfToken()
```

Comment:
The first call returns error as well, shouldn't we check it whether is nil or not?

Reply:
Ok but what should we do if it is nil?